example/tailored/meta/index.js

```diff
@@ -0,0 +1,8 @@
+const example = require('./extractors/example-tailored-meta-data-extractor');
+
+/**
+ * NOTE: order of execution bound to position in the array
+ */
+module.exports = [
+    example
+]
```

Comment:
Simple question : is this to be able to use multiple extractor configurations ?

Reply:
It exports an array, so yes! You can dump "as many" as you want.

Reply:
These extractors will be run once on creation of the tailored meta and their output dumped into a dedicated tailored additions ttl! :)